src/test/components/Quote.test.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import Quote from '../../components/Quote/Quote';
+
+it('Quote component', () => {
+  const tree = render(<Quote />);
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] (Required) Wonderful work done so far. It would be nice to use `React Testing Library` to test the functionality of your `React Components` and not just use only `Jest Snapshots` to create Snapshots. You can check this [_**Link**_](https://testing-library.com/docs/react-testing-library/intro/) for knowledge on how to use React Testing Library to test components

Reply:
Done. I added to Calculator component because there's no logic inside Quote component